task_4/app.js

```diff
@@ -0,0 +1,50 @@
+// Create a car object, add a color property to it with the value equals 'black'
+const car = new Object()
+
+// Change the color property of the car object to 'green'
+car.color = 'green'
+
+//Add the power property to the car object, which is a function and displays the engine power to the console
+car.power = function (){ console.log('Horse power: 665')}
+car.power()
+
+// Pears and apples are accepted to the warehouse, write a function that returns the result of adding the number of accepted pears and apples
+const incomingInventory = (pear, apple) => {
```

Comment:
In the task, you just need to display the amount, in your function the amount is not calculated anywhere.